dmoj/settings.py

```diff
@@ -65,6 +65,8 @@
 DMOJ_PROBLEM_MIN_MEMORY_LIMIT = 0  # kilobytes
 DMOJ_PROBLEM_MAX_MEMORY_LIMIT = 1048576  # kilobytes
 DMOJ_PROBLEM_MIN_PROBLEM_POINTS = 0
+DMOJ_PROBLEM_MIN_USER_POINTS_VOTE = 1
```

Comment:
Can we leave a short comment above these explaining what they are? It wasnt obvious to me from just the name before reading further in the patch. One sentence would be fine.